### ISSUE TITLE
Resync all - improve process

### ISSUE THREAD

**Issue by Rahe** [BeAPI/bea-content-sync-fusion#73]
At this time, when executing the `resync site` or `resync all` CLI command, the queries are executed on every website and then on every content.

So if I have Site 1 with attachments, just say 10, that are synced to all the sites of the network and having 100 sites (already synced with the medias) I will play the hook 10 times on every blog.

Don't we have to launch the resync only on the emitter sites ?
This can be hard to handle because we can use multi post_Type and multi taxonomies etc. But if the web isn't in any emiter, then we can skip it ?

**Comment by herewithme** [comment]
You're right, I used this approach because sync registerations may vary by site/network.

I wonder if I should not continue to pass on each site, but start by looking if "locally", there is an active sync.

**Comment by asadowski10** [comment]
Yes you are clearly right !!!

**Comment by herewithme** [comment]
@Rahe I implemented a first check in the site () method, it seems to do the job. An opinion?

**Comment by Rahe** [comment]
@herewithme Not working for me, for the "all except emitter" param there is still some results.
